src/api/spec/features/beta/webui/users/notifications_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'browser_helper'
+
+RSpec.describe 'User notifications', type: :feature, js: true do
+  let!(:user) { create(:confirmed_user, :in_beta) }
+
+  describe 'when having no notifications' do
+    context 'when accessing the notifications page' do
+      before do
+        login user
+        visit my_notifications_path
+      end
+
+      it 'shows no notifications' do
+        expect(page).to have_text('There are no notifications')
+      end
+    end
+  end
+
+  describe 'when having notifications' do
+    let!(:notification_for_projects_comment) { create(:web_notification, :comment_for_package, subscriber: user) }
+    let!(:another_notification_for_projects_comment) { create(:web_notification, :comment_for_package, subscriber: user) }
+    let(:notifiable) { notification_for_projects_comment.notifiable }
+    let(:another_notifiable) { another_notification_for_projects_comment.notifiable }
+    let(:project) { notifiable.commentable.project }
+
+    context 'when clicking on the Comments filter' do
+      before do
+        login user
+        visit my_notifications_path
+      end
+
+      it 'shows all unread comment notifications' do
+        find(id: '#notifications-dropdown-trigger').click if mobile?
+        within('#filters') { click_link('Comments') }
+        expect(page).to have_text(notifiable.commentable.name)
+      end
+
+      context 'when marking a comment notification as read' do
+        before do
+          find(id: '#notifications-dropdown-trigger').click if mobile?
+          within('#filters') { click_link('Comments') }
+          find_link(id: format('update-notification-%d', notification_for_projects_comment.id)).click
```

Comment:
This should work, right?
```suggestion
          click_link("update-notification-#{notification_for_projects_comment.id}")
```

Reply:
Yyyesss